src/application.coffee

```diff
@@ -36,6 +36,9 @@ logTypes =
 	downloadApp:
 		eventName: 'Application download'
 		humanName: 'Downloading application'
+	downloadAppDelta:
+		eventName: 'Application delta download'
```

Comment:
I just merged this :/ 
I was thinking if it just says "Application download" we'll know it's a regular download from now on... but I guess we could PR that